wolfssh/internal.h

```diff
@@ -341,10 +341,10 @@ enum {
     #define DEFAULT_HIGHWATER_MARK ((1024 * 1024 * 1024) - (32 * 1024))
 #endif
 #ifndef DEFAULT_WINDOW_SZ
-    #define DEFAULT_WINDOW_SZ 16384
+    #define DEFAULT_WINDOW_SZ (128 * 1024)
 #endif
 #ifndef DEFAULT_MAX_PACKET_SZ
-    #define DEFAULT_MAX_PACKET_SZ (16 * 1024)
+    #define DEFAULT_MAX_PACKET_SZ 32768
```

Comment:
A comment on the choice of size should be added here, something to the effect of default max packet size defined in RFC XXX